BuildAddOn.py

```diff
@@ -242,7 +249,7 @@ def Check7ZInstallation ():
         raise Exception ('7Zip not installed!')
 
 
-def CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, configuration, languageCode=None, isRelease=False):
+def CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, configuration, languageCode=None):

```

Comment:
When can the `languageCode` be `None`? Why don't we use the default language code in this case?

Reply:
Fixed it everywhere